src/Controller/EnMarche/EventController.php

```diff
@@ -173,4 +174,28 @@ public function invitationSentAction(Request $request, Event $event): Response
             'invitations_count' => $invitationsCount,
         ]);
     }
+
+    /**
+     * @Route("/desinscription", name="app_event_unregistration", condition="request.isXmlHttpRequest()")
+     * @Method("GET|POST")
+     */
+    public function unregistrationAction(Request $request, Event $event): JsonResponse
+    {
+        if (!$this->isCsrfTokenValid('citizen_action.unregistration', $token = $request->request->get('token'))) {
```

Comment:
what about renaming `citizen_action.unregistration` on `event.unregistration` for both CitizenAction and Event unsubscriptions? (as CitizenAction is an Event too)